docs/CountMinSketch_Commands.md

```diff
@@ -24,27 +58,132 @@ O(1)
 
 OK on success, error otherwise
 
-## CMS.INITBYPROB
+#### Reserve example
```

Comment:
Not a `Reserve` command - you might just want to go with a generic `Example` header.

Reply:
Changed to generic example